src/pages/Messages/_Star.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import Tooltip from '../../components/Tooltip'
+import { Button } from 'antd'
+import {deleteStar, unstarMessage, starMessage, submitStar} from "../../actions/starMessage"
+
+const Star = ({currentUser, isStarred, toggleStar}) => {
+  const handleStarClick = (event) => {
+    if(event){ event.preventDefault() }
+    toggleStar()
+  }
+  return (
+
+    <Tooltip placement='top' title="Star this message">
+      <Button
+        icon={isStarred ? 'star' : 'star-o'}
+        style={isStarred ? {color: '#ffd700'} : {color: '#000000'}}
+        onClick={handleStarClick}
+        />
+    </Tooltip>
+
+  )
+}
+
+const mapStateToProps = () => ({
+})
+
+const mapDispatchToProps = (dispatch, {isStarred, messageId}) => ({
+  toggleStar: () => (
+      isStarred ? dispatch(unstarMessage(messageId)) & dispatch(deleteStar(messageId)) : dispatch(starMessage(messageId)) & dispatch(submitStar(messageId))
+  )
+})
+
+
+Star.displayName = 'Star'
+
+
```

Comment:
Same as above, move the `displayName` line above `mapStateToProps` and use only one empty line between these.